imports/ui/game-sheet/GameSheet.js

```diff
@@ -0,0 +1,260 @@
+import Players from '../../api/Players';
+
+import Constants from '../../api/Constants';
+import EloCalculator from '../../api/EloCalculator';
+import GameRecordUtils from  '../../api/utils/GameRecordUtils';
+
+import './GameSheet.html'
+
+/**
+ * Retreives the appropriate start points for a game
+ * @param[in] style The game style to get the start points for
+ *
+ * @returns {Number} The starting points for a mahjong game
+ */
+function _getStartPoints(style) {
+    switch (style) {
+    case Constants.GAME_TYPE.HONG_KONG:
+	return Constants.HKG_START_POINTS;
+    case Constants.GAME_TYPE.JAPANESE:
+	return Constants.JPN_START_POINTS;
+    }
+};
+
+/**
+ * Template predicate to determine if game is Hong Kong style
+ * @param[in] style The game style
+ *
+ * @returns {Boolean} Whether the game is Hong Kong style
+ */
+function _isHongKong(style) {
+    return style === Constants.GAME_TYPE.HONG_KONG;
+};
+
+/**
+ * Template predicate to determine if game is Japanese style
```

Comment:
@amrocha Do you think I'm overdoing the function documentation?

Reply:
Hmm yeah I think for like, self-evident function like this one it would be fine to not have anything, but you have type information there which really helps in vanilla JS so I think it adds something

Reply:
I have no idea how useful type information is.  I commented some of the functions in detail, and some I left uncommented.  Which is better?

Reply:
Yeah, I'm not going to be around forever so I think it might be important.  Do you think it takes too much away?

Reply:
Give me a day or two, I'm gonna download it and try looking at it without as much documentation.

Reply:
With some editor setups you should be able to collapse comments